### PR TITLE
[BF] fix sidecars suggested by Sam

### PR DESCRIPTION
**Pull request by arnaudbore**
Thank you @SamGuay

### CODE REVIEW

**Comment by arnaudbore** [comment]
Fix also #242 and #241 :)

File: docs/how-to/use-advanced-commands.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# How to use advanced configuration
+# Advanced configuration and command
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
# Advanced configuration and commands
```

File: docs/how-to/use-advanced-commands.md [hunk 2]
```diff
@@ -43,7 +45,7 @@ same level as the `"description"` entry.
 }
 ```
 
-## custom_entities combined with extractors
+### custom_entities combined with extractors
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `custom_entities` combined with extractors
```

File: docs/how-to/use-advanced-commands.md [hunk 3]
```diff
@@ -57,15 +59,15 @@ entities directly into the final filename. custom_entities can be a list that
 combined extractor keys and regular entities. If key is `task` it will
 automatically add the field "TaskName" inside the sidecase file.
 
-## search_method
+### search_method
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `search_method`
```

File: docs/how-to/use-advanced-commands.md [hunk 4]
```diff
@@ -57,15 +59,15 @@ entities directly into the final filename. custom_entities can be a list that
...
 
 default: `"search_method": "fnmatch"`
 
 fnmatch is the behaviour (See criteria) by default and the fall back if this
 option is set incorrectly. `re` is the other choice if you want more flexibility
 to match criteria.
 
-## dup_method
+### dup_method
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `dup_method`
```

File: docs/how-to/use-advanced-commands.md [hunk 5]
```diff
@@ -77,14 +79,14 @@ customEntities of the other acquisitions. This behavior is a
 [heudiconv](https://heudiconv.readthedocs.io/en/latest/changes.html) inspired
 feature.
 
-## case_sensitive
+### case_sensitive
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `case_sensitive`
```

File: docs/how-to/use-advanced-commands.md [hunk 6]
```diff
@@ -77,14 +79,14 @@ customEntities of the other acquisitions. This behavior is a
...
 
 default: `"case_sensitive": "true"`
 
 If false, comparisons between strings/lists will be not case sensitive. It's
 only disabled when used with `"search_method": "fnmatch"`.
 
-## post_op
+### post_op
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `post_op`
```

File: docs/how-to/use-advanced-commands.md [hunk 7]
```diff
@@ -130,23 +132,23 @@ which datatype is fmap.
 Finally, this is a template string and dcm2bids will replace `src_file` and
 `dst_file` by the source file (input) and the destination file (output).
 
-## dcm2niixOptions
+### dcm2niixOptions
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `dcm2niixOptions`
```

File: docs/how-to/use-advanced-commands.md [hunk 8]
```diff
@@ -130,23 +132,23 @@ which datatype is fmap.
...
 
 default: `"dcm2niixOptions": "-b y -ba y -z y -f '%3s_%f_%p_%t'"`
 
 Arguments for dcm2niix
 
-## compKeys
+### compKeys
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `compKeys`
```

File: docs/how-to/use-advanced-commands.md [hunk 9]
```diff
@@ -130,23 +132,23 @@ which datatype is fmap.
...
 
 default: `"compKeys": ["SeriesNumber", "AcquisitionTime", "SidecarFilename"]`
 
 Acquisitions are sorted using the sidecar data. The default behaviour is to sort
 by `SeriesNumber` then by `AcquisitionTime` then by the `SidecarFilename`. You
 can change this behaviour setting this key inside the configuration file.
 
-## criteria
+### criteria
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `criteria`
```

File: docs/how-to/use-advanced-commands.md [hunk 10]
```diff
@@ -245,7 +247,7 @@ command.
 
     ```
 
-## --auto_extract_entities
+### --auto_extract_entities
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `--auto_extract_entities`
```

File: docs/how-to/use-advanced-commands.md [hunk 11]
```diff
@@ -310,7 +312,7 @@ like this.
 :radioactive: You can find more detailed information by looking at the file [`dcm2bids/utils/utils.py`](../dcm2bids/utils/utils/) and
 more specifically *`auto_extractors`* and *`auto_entities`* variables.
 
-## --bids_validate
+### --bids_validate
```
Comment (SamGuay) [inline docs/how-to/use-advanced-commands.md]:
```suggestion
### `--bids_validate`
```

**Review by SamGuay — APPROVED** [review]

**Pull request merged**